modules/core/04-channel/types/upgrade_test.go

```diff
@@ -4,13 +4,13 @@ import (
 	errorsmod "cosmossdk.io/errors"
 
 	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
-	"github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
```

Comment:
typically we allow the `types` directory to be used without a prefix when we are in that package or sub package. So `types` in the 04-channel subdirectory would be just `types`, but I'm happy to be more explicit as well!

Reply:
happy to change back, think I just picked one that was in the merge conflicts but I can just use `types`